test/customer_test.rb

```diff
@@ -5,29 +5,28 @@
 class CustomerTest < MiniTest::Test
   attr_reader :parent, :customer
   def setup
-    sales_engine = SalesEngine.new.startup
-    parent = CustomerRepository.new('test/support/sample_customers.csv', sales_engine)
+    sales_engine = SalesEngine.new
+    parent = CustomerRepository.new('test/support/customers.csv', sales_engine)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.